python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -537,16 +570,32 @@ class RandomForestClassifier(Base):
         return self
 
     def _predict_model_on_gpu(self, X, output_class,
-                              threshold, algo, num_classes):
-        _, _, n_rows, n_cols, X_dtype = \
-            input_to_dev_array(X, order='C', check_dtype=self.dtype,
-                               check_cols=self.n_cols)
+                              threshold, algo,
+                              num_classes):
 
-        treelite_model = self._get_treelite(num_features=n_cols,
-                                            task_category=num_classes)
+        cdef ModelHandle cuml_model_ptr = NULL
+        _, _, n_rows, n_cols, X_type = \
+            input_to_dev_array(X, order='C')
```

Comment:
Since you are not storing the actual array here, is order='C' the correct order to ask? Just want to be sure we are not transforming `X`'s order unnecessarily

Reply:
The order does not matter, removed the order type.